source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "rake db:schema:load vs rake db:migrate"
+date: 2016-03-23 08:09:25 +0100
+comments: true
+author: Ivica Lakatoš
+categories:
+  - Ruby and Rails
+tags:
+  - ruby
+  - rails
+  - rake task
+  - globalize
+published: true
+---
+
+Sooner or later every new ruby developer need to understand  differences between this two common rake task. Basically, simple definition tell us everything we need to know:
+
++ `rake db:migrate` runs migrations that have not run yet
++ `rake db:schema:load` loads the schema.db file into database.
+
+but the right question is when to use one or the other.
+
+**Advice:** <a id="advice"></a> when you adding new migration to a existing app then you need to run `rake db:migrate`, but when you join to existing application (_especially some old application_), or when you drop your applications database and you need to create it again, always run `rake db:schema:load` to load schema.
```

Comment:
@ikera I think that this line should be "when you **are** adding **a** new migration to **an** existing app..."